src/main/daml/Daml/Finance/Interface/Types/Date/DayCount.daml

```diff
@@ -9,24 +9,24 @@ module Daml.Finance.Interface.Types.Date.DayCount
 data DayCountConventionEnum
   = Act360
   -- ^ Actual 360. In CDM it is called *DayCountFractionEnum_ACT_360*. In ISO20022 it is called *A004*.
-  | Act365_Fixed
+  | Act365Fixed
   -- ^ Actual 365 fixed. In CDM it is called *DayCountFractionEnum_ACT_365_FIXED*. In ISO20022 it is called *A005*.
   | Act365L
   -- ^ Actual 365L. In CDM it is called *DayCountFractionEnum_ACT_365L*. In ISO20022 it is called *A009*.
-  | ActAct_AFB
+  | ActActAFB
   -- ^ Actual Actual AFB. In CDM it is called *DayCountFractionEnum_ACT_ACT_AFB*. In ISO20022 it is called *A010*.
-  | ActAct_ISDA
+  | ActActISDA
   -- ^ Actual Actual ISDA. In CDM it is called *DayCountFractionEnum_ACT_ACT_ISDA*. In ISO20022 it is called *A008*.
-  | ActAct_ICMA
+  | ActActICMA
   -- ^ Actual Actual ICMA. In CDM it is called *DayCountFractionEnum_ACT_ACT_ICMA* and *DayCountFractionEnum_ACT_ACT_ISMA* (they are identical: https://www.isda.org/2011/01/07/act-act-icma/). In ISO20022 it is called *A006*. Also called ISMA in the 1998 ISDA paper.
-  | Basis_1_1
+  | Basis11
   -- ^ 1/1. In CDM it is called *DayCountFractionEnum__1_1*. Currently not included in ISO20022.
-  | Basis_30360
+  | Basis30360
   -- ^ 30/360. In CDM it is called *DayCountFractionEnum__30_360*. In ISO20022 it is called *A001*. Also called 30/360 ISDA or American Basic rule.
-  | Basis_30360_ICMA
+  | Basis30360ICMA
   -- ^ 30/360 ICMA. In CDM it is called *DayCountFractionEnum__30E_360*. In ISO20022 it is called *A011*. Also called Basic Rule. This corresponds to "30E/360" of the 2006 ISDA definitions.
-  | Basis_30E360
+  | Basis30E360
   -- ^ 30E/360. In CDM it is called *DayCountFractionEnum__30E_360_ISDA*. In ISO20022 it is called *A007*. Also called Eurobond basis. This corresponds to "30E360 (ISDA)" of the 2006 ISDA definitions.
-  | Basis_30E3360
+  | Basis30E3360
   -- ^ 30E3/360. Currently not included in CDM. In ISO20022 it is called *A013*. Also called Eurobond basis model 3.
   deriving (Eq, Show)
```

Comment:
I think most of these look better now, except Basis11, which is confusing. Not sure how to fix it with the new convention though, maybe Basis1 (since it is always 1.0) or BasisOneOne?
My vote probably goes to Basis1.

Reply:
I'll change it to Basis1